.github/workflows/molecule.yaml

```diff
@@ -19,9 +19,6 @@ jobs:
             pip install -r requirements-testing.txt
             ansible-galaxy install -r requirements.yml
             ansible-galaxy install -r molecule/requirements.yml
-        - name: Image mirroring
-          run: |
-            sed -f vm-testing/dev-images.sed  -i roles/tas_single_node/defaults/main.yml
```

Comment:
You can probably remove this file, right? https://github.com/securesign/artifact-signer-ansible/blob/main/vm-testing/dev-images.sed

Reply:
I will keep that script there because it is still referenced in doc for manual testing.

I modified molecule doc to remove that step.